### PR TITLE
gsctl reference main page added

### PR DESCRIPTION
**Pull request by marians**
This change adds the "reference" main menu item, linking to /reference/gsctl/ only for now.

RFR @puja108

### CODE REVIEW

**Comment by marians** [comment]
Related: https://github.com/giantswarm/docs/pull/191

File: content/reference/gsctl/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+# `gsctl` Reference
+
+## Command Reference {#commands}
+
+- `create`: Create things, like kubectl configuration, or key-pairs
+- `info`: Print some information
+- `list`: List things, like organizations, clusters, key-pairs
+- `login`: Sign in as a user
+- `logout`: Sign out the current user
```
Comment (puja108) [inline content/reference/gsctl/index.md]: I think `out` should be in the end of the sentence

File: content/reference/gsctl/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+- `ping`: Check API connection
+- `version`: Print version number
+
+Check `gsctl <command> --help` for details.
+
+## Installing and Updating {#install}
+
+`gsctl` comes as a self-contained binary for Mac, Linux, and Windows. Below you find installation instructions for different platforms. If you want to build `gsctl` from source, find everything you need in it's [GitHub repository](https://github.com/giantswarm/gsctl).
```
Comment (puja108) [inline content/reference/gsctl/index.md]: `it's` -> `its`

File: content/reference/gsctl/index.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+tar xzf gsctl-$VERSION-darwin-amd64.tar.gz
+sudo cp gsctl-$VERSION-darwin-amd64/gsctl /usr/local/bin/
+```
+
+### Linux {#install-linux}
+
+Execute the commands below to install the latest released version. The same commands work for upgrading from a previously installed version.
+
+```nohighlight
+curl -O http://downloads.giantswarm.io/gsctl/0.1.0/gsctl-0.1.0-linux-amd64.tar.gz
```
Comment (puja108) [inline content/reference/gsctl/index.md]: why don't we use the nifty $version from above here?

File: content/reference/gsctl/index.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+tar xzf gsctl-0.1.0-linux-amd64.tar.gz
+sudo cp gsctl-0.1.0-linux-amd64/gsctl /usr/local/bin/
+```
+
+### Windows {#install-win}
+
+- Download the [64 bit](http://downloads.giantswarm.io/gsctl/0.1.0/gsctl-0.1.0-windows-amd64.zip) or [32 bit](http://downloads.giantswarm.io/gsctl/0.1.0/gsctl-0.1.0-windows-386.zip) version of `gsctl` for Windows
+- Copy the contained `gsctl.exe` to a convenient location
```
Comment (puja108) [inline content/reference/gsctl/index.md]: and add it to your PATH?

File: content/reference/gsctl/index.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+- Not all API responses (error cases) are handled properly. If you experience a "Unknown Status code" or similar error message, please report this [in an issue](https://github.com/giantswarm/gsctl/issues/new). Thanks!
+- Check our [issues](https://github.com/giantswarm/gsctl/issues?q=is%3Aopen+is%3Aissue+label%3Akind%2Fbug) for more
+
+## Changelog {#changelog}
+
+You'll find info on changes in the [release description](https://github.com/giantswarm/gsctl/releases).
+
+## Feedback {#feedback}
+
+We welcome your feedback on `gsctl`. If you feel like sharing openly, use the GitHub repository and create an [issue](https://github.com/giantswarm/gsctl/issues), so other users can participate. Otherwise please use the known support channels.
```
Comment (puja108) [inline content/reference/gsctl/index.md]: maybe add `Giant Swarm` before `support channels` and change `known` to `common`?

File: content/reference/gsctl/index.md (hunk 4 above)
Comment (marians) [inline content/reference/gsctl/index.md]: Changed to:
> Copy the contained `gsctl.exe` to a convenient location that's in your `%PATH%`, or add the `gsctl` location to your `%PATH%`

File: content/reference/gsctl/index.md (hunk 1 above)
Comment (marians) [inline content/reference/gsctl/index.md]: Changed, also in `gsctl` usage

File: content/reference/gsctl/index.md (hunk 3 above)
Comment (marians) [inline content/reference/gsctl/index.md]: Right, forgot. Now added.

File: content/reference/gsctl/index.md (hunk 5 above)
Comment (marians) [inline content/reference/gsctl/index.md]: done

**Pull request merged**